### PR TITLE
Upgraded parsec-tool to 0.3.0

### PR DESCRIPTION
**Pull request by ygoyal18**

### CODE REVIEW

**Review by esajaa — DISMISSED** [review]

**Review by JanneKiiskila — CHANGES_REQUESTED** [review]
This one leaves freely hanging on main, we need to use a tag or hash so that the builds remain repeatable.

**Comment by JanneKiiskila** [comment]
Filename should also match the version being pulled in.

**Comment by ygoyal18** [comment]
@JanneKiiskila The file name includes the version which needs to be pulled in. The branch=main is mentioned and SRC_REV=0.3.0 will be populated from file name.

File: recipes-misc/parsec-tool/parsec-tool_0.3.0.bb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 inherit cargo
 
-SRC_URI = "git://github.com/parallaxsecond/parsec-tool;rev=bae78ec41e03540ba5e96b1d4980f3445ab7d7af;protocol=https;branch=main"
+SRC_URI = "git://github.com/parallaxsecond/parsec-tool;protocol=https;branch=main"
```
[the remark this answers is not in the document]
Comment (ygoyal18) [reply recipes-misc/parsec-tool/parsec-tool_0.3.0.bb:3]: Ahh okay. I will add a tag.

Comment (ygoyal18) [inline recipes-misc/parsec-tool/parsec-tool_0.3.0.bb:3]:
```suggestion
SRC_URI = "git://github.com/parallaxsecond/parsec-tool;protocol=https;branch=main;tag=${PV}"
SRCREV = "${PV}"
```

**Review by x85446 — APPROVED** [review]
LGTM

**Comment by JanneKiiskila** [comment]
Ah, yes, that PV variable thing is more clever.

**Pull request merged**